frontend/src/component/common/NewConstraintAccordion/ConstraintAccordionEdit/ConstraintAccordionEditBody/RestrictiveLegalValues/RestrictiveLegalValues.tsx

```diff
@@ -50,6 +50,17 @@ export const getIllegalValues = (
     return constraintValues.filter((value) => deletedValuesSet.has(value));
 };
 
+const StyledValuesContainer = styled('div')(({ theme }) => ({
+    display: 'flex',
+    flexWrap: 'wrap',
+    gap: theme.spacing(1),
+    padding: theme.spacing(2),
+    border: `1px solid ${theme.palette.divider}`,
+    borderRadius: theme.shape.borderRadiusMedium,
+    maxHeight: '378px',
```

Comment:
where does this specific value come from?

Reply:
Nicolae and the ticket